docs/GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -54,194 +24,171 @@ export COBALT_PIPELINE_NAME="Cobalt CICD Pipeline"
-        * Click [+/Add] and select Add role assignment
-            * From the sub-menu, select 'Owner' as a role from the drop down and search for the newly created Service Principal (i.e. cobalt-hw-admin-sp-`<username>` or cobalt-az-iso-admin-sp-`<username>`)
+        1. Filter for subscriptions and navigate to the subscriptions list
+        2. Either choose a subscription or create a new one (ex. Cobalt-Contoso-Deployments)
+        3. Select your chosen subscription then select the Access control (IAM) tab from the menu blade.
+        4. Click [+/Add] and select Add role assignment
+        5. From the sub-menu, select 'Owner' as a role from the drop down and search for the newly created Service Principal (i.e. cobalt-hw-admin-sp-`<username>` or cobalt-az-iso-admin-sp-`<username>`)
 
-                ![Role Assignment menu](https://user-images.githubusercontent.com/31149154/63708249-7ed23400-c7f9-11e9-8dbb-c15dcdaf3a37.png)
+            ![Role Assignment menu](https://user-images.githubusercontent.com/31149154/63708249-7ed23400-c7f9-11e9-8dbb-c15dcdaf3a37.png)
 
-            * Click Save
+        6. Click Save
 
     * Create Resource Group and Storage Account for backend state
-        * Filter for Storage accounts and navigate to the storage account list
-        * Click [+/Add] and enter values for the following fields:
+        1. Filter for Storage accounts and navigate to the storage account list
+        2. Click [+/Add] and enter values for the following fields:
             * Subscription: Your preferred enterprise subscription for Cobalt template deployments
             * Resource group: Create new (ex. cobalt-devint-hw-admin-rg or cobalt-devint-az-iso-admin-rg )
             * Storage account name: (ex. cobalttfstates)
-        * Click [Review+Create] then [Create]
-        * Once deployment for storage account is completed, go to the resource and visit the Blobs sub-menu
-        * Click [+Container] then create a container name (ex. az-hw-remote-state-container or az-iso-remote-state-container) with private access
-
-> The following CLI command(s) can be run as an alternative to using the portal-based instructions. Note this creates the Service Connection within Azure DevOps, but you will still need to setup the Service Principal:
-
-```bash
-az devops service-endpoint azurerm create --azure-rm-subscription-id $SUBSCRIPTION_ID --azure-rm-subscription-name "$SUBSCRIPTION_NAME" --azure-rm-tenant-id $TENANT_ID --azure-rm-service-principal-id $SERVICE_PRIN_ID --name "$SERVICE_CONN_NAME"
-```
+        3. Click [Review+Create] then [Create]
+        4. Once deployment for storage account is completed, go to the resource and visit the Blobs sub-menu
+        5. Click [+Container] then create a container name (ex. az-hw-remote-state-container or az-iso-remote-state-container) with private access
 
 3. **Configure Azure Devops pipeline using Azure resource values**
```

Comment:
Can you provide some more context around `Azure resource values`?

Reply:
This was step 3. I expanded the summary of this step.